src/shared/utils.ts

```diff
@@ -1,3 +1,4 @@
+import { postgresReservedKeywords } from '../constants';
```

Comment:
We can now move `constants` to `shared/constants` if you so desire.

Reply:
but I guess without exporting at `shared/index.ts`?

Reply:
Hmm, yes. I have some regrets about exporting everything from index files.